### DISCUSSION TITLE
Diehl & Cook Network Parameters Values

### DISCUSSION THREAD

**Discussion by GutiPerez** [BindsNET/bindsnet#508] [Q&A] [answered]
Good afternoon

I would like to know how you have made the decision on the parameter values of the Diehl & Cook network. I am referring to parameters such as the initial values of the inhibitory and excitatory layers, theta_plus, normalization value, pre and pro synaptic learning rates (nu), etc. My main curiosity is whether you have taken such values from the original code (in Brian) or whether they have been choosen after an extensive testing.

Thanks in advance.

**Comment by Hananel-Hazan** [comment, marked as answer]
Because the implementation in BindsNET is not the exact replication of the original code, the values in the BindsNET examples are the results of an extensive testing base on the original parameters.

**Reply by GutiPerez** [reply]
Got it. Then, thanks for your work and for your answer.

Regards,
Miguel
